.travis.yml

```diff
@@ -24,14 +24,49 @@ branches:
 
 before_install:
   - nvm install --latest-npm
+  - |
+    if [[ "$WORDPRESS" = "true" ]]; then
+      # upgrade docker-compose
+      sudo rm /usr/local/bin/docker-compose
+      curl -L https://github.com/docker/compose/releases/download/1.24.0/docker-compose-`uname -s`-`uname -m` > docker-compose
+      chmod +x docker-compose
+      sudo mv docker-compose /usr/local/bin
+
+      # Check out WordPress
+      git clone git://develop.git.wordpress.org/ wordpress
```

Comment:
I don't like the fact that we're relying on the local environment of an external repository to achieve that. Especially since it's not written to be reused outside of Core as a reusable "module/package/script". 

Can't we instead, just have the docker-compose file here in this repo too (copied).
Also, I kind of like  the fact that when you open `package.json` you see the scripts `env:start`.... I feel we should have these scripts in this repo tooo.